gulp/tasks/test.js

```diff
@@ -95,6 +95,9 @@ function _getProtractorParams() {
   // If --version=number flag is added on the command-line.
   params = _addCommandLineFlag( params, commandLineParams, 'version' );
 
+  // If --suite=suite1,suite2 flag is added on the command-line.
+  params = _addCommandLineFlag( params, commandLineParams, 'suite' );
+
   return params;
```

Comment:
Not a huge deal, but can we do something like `_addCommandLineFlags` (plural) here, that passes in 'version', 'suite', & all the other flags?